### PR TITLE
Update dependencies (minors + patches)

### PR DESCRIPTION
**Pull request by joanlopez**
It updates few dependencies:
- Closes https://github.com/friendsofgo/killgrave/pull/56
- Closes https://github.com/friendsofgo/killgrave/pull/55
- Closes https://github.com/friendsofgo/killgrave/pull/48
- Closes https://github.com/friendsofgo/killgrave/pull/46

Additionally, it also updates the project's Go version.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,16 +1,16 @@
 module github.com/friendsofgo/killgrave
 
-go 1.13
+go 1.15
```
Comment (aperezg) [inline go.mod:3]: why this change? we're introducing  something that's is only working on go 1.15.

Furthermore you should updgrade the `.circleci/config.yml` to build and testing on 1.14 and 1.15

Comment (joanlopez) [reply go.mod:3]: Okay, thanks for the suggestion! 🙏  I'm gonna update the Go versions defined on the CI configuration 🚀
